packages/core/test/test-util/test-services/v4/test-service/TestComplexType.ts

```diff
@@ -62,16 +62,6 @@ export interface TestComplexType {
    * @nullable
    */
   floatProperty?: number;
-  /**
-   * Time Property.
-   * @nullable
-   */
-  timeProperty?: Time;
```

Comment:
I know there is a task to remove invalid data type in v4, but maybe we can make a separate pr.

Reply:
I started on a separate PR.